### ISSUE TITLE
Can't delete past queries with the mouse.

### ISSUE THREAD

**Issue by Pauan** [Pauan/tab-organizer#72]
Originally reported on Google Code with ID 71

```
What steps will reproduce the problem?
1. Open the past queries list.
2. Try to delete a query.

What is the expected output? What do you see instead?
Can't. Should be able to.

Please provide any additional information below.
This is as simple as placing a close button on the right side of each item, on hover.
```

Reported by `pcxunlimited` on 2010-10-03 13:11:23

**Comment by Pauan** [comment]
```
This issue was updated by revision 88fe1fb49a.
```

Reported by `pcxunlimited` on 2010-10-11 05:42:23
- **Status changed**: `Pending`

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-10-17 17:04:03
- **Status changed**: `Verified`

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:44:48
- **Status changed**: `Fixed`
